### PR TITLE
ForwardCommandController declares parameters

### PR DESCRIPTION
**Pull request by bmagyar**
Tested against demo repo and also in conjunction with joint trajectory controller as well as diff drive controller (I hacked the demo system locally a bit ;))

### CODE REVIEW

File: forward_command_controller/src/forward_command_controller.cpp [hunk 1]
```diff
@@ -34,31 +34,46 @@ ForwardCommandController::ForwardCommandController()
   joints_command_subscriber_(nullptr)
 {}
 
+controller_interface::return_type
+ForwardCommandController::init(
+  const std::string & controller_name)
```
Comment (destogl) [inline forward_command_controller/src/forward_command_controller.cpp:39]: One line please

```suggestion
ForwardCommandController::init(const std::string & controller_name)
```

File: forward_command_controller/src/forward_command_controller.cpp [hunk 2]
```diff
@@ -34,31 +34,46 @@ ForwardCommandController::ForwardCommandController()
...
+{
+  auto ret = ControllerInterface::init(controller_name);
+  if (ret != controller_interface::return_type::SUCCESS) {
+    return ret;
+  }
+
+  try {
```
Comment (destogl) [inline forward_command_controller/src/forward_command_controller.cpp:46]: Do we have to declare Paramus in the Init step, or can this be done in the configure step? (If in configure, maybe it doesn’t work for PAL use-case)

**Review by destogl — APPROVED** [review]
Just check if configure can be used...

File: forward_command_controller/src/forward_command_controller.cpp (hunk 2 above)
Comment (bmagyar) [reply forward_command_controller/src/forward_command_controller.cpp:46]: All PAL ported controllers declared them in init and I'd like to think I'm familiar with their use case. It should be fine as-is

**Pull request merged**